src/styles/stars.scss

```diff
@@ -0,0 +1,27 @@
+@use './variables.scss' as *;
+
+.stars {
+  width: 96px;
```

Comment:
The width of the .stars container is hardcoded as 96px. Consider using a more flexible unit or calculating it based on the number of stars and their width to make it more adaptable.